src/main/java/graphql/nadel/result/ResultComplexityAggregator.java

```diff
@@ -0,0 +1,45 @@
+package graphql.nadel.result;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class ResultComplexityAggregator {
+    private int totalNodeCount = 0;
+    private LinkedHashMap serviceNodeCounts = new LinkedHashMap<String, Integer>();
```

Comment:
Again please use generic interfaces for collections with specifics on the RHS

Also always use generics where you can


`private Map<String, Integer> serviceNodeCounts = new LinkedHashMap<>();`

You should have got little yellow squiggly lines from IDEA as a warning here on both cases